features/0160-connection-protocol/README.md

```diff
@@ -375,15 +375,15 @@ The signature data must be used to verify against the invitation's `recipientKey
 
 - The `@type` attribute is a required string value that denotes that the received message is a connection request.
 - The `~thread` block contains a `thid` reference to the `@id` of the request message. 
-- The `connection` attribute contains the `did` and `did_doc` attributes to enable simpler signing.
-- The `did` attribute is a required string value and denotes DID in use by the _inviter_. Note that this may not be the same DID used in the invitation.
-- The `did_doc` attribute contains the associated DID Doc. If the DID method for the presented DID is not a peer method and the DID Doc is resolvable on a ledger, the `did_doc` attribute is optional.
+- The `connection` attribute contains the `DID` and `DIDDoc` attributes to enable simpler signing.
+- The `DID` attribute is a required string value and denotes DID in use by the _inviter_. Note that this may not be the same DID used in the invitation.
+- The `DIDDoc` attribute contains the associated DID Doc. If the DID method for the presented DID is not a peer method and the DID Doc is resolvable on a ledger, the `DIDDoc` attribute is optional.
 
 In addition to a new DID, the associated DID Doc might contain a new endpoint. This new DID and endpoint are to be used going forward in the connection.
 
 #### Response Transmission
 
-The message should be packaged in the wire level format, using the keys from the request, and the new keys presented in the internal did doc. 
+The message should be packaged in the wire level format, using the keys from the request, and the new keys presented in the internal DID Document.
```

Comment:
Hey, just a quick note that in the DID core spec we recently decided to change the capitalization of all key terms to lower case (e.g. "DID document" instead of "DID Document"), see https://github.com/w3c/did-core/pull/73/
```suggestion
The message should be packaged in the wire level format, using the keys from the request, and the new keys presented in the internal DID document.
```

Reply:
Updating those references might be a bit out of scope for this PR but I went ahead and changed them (including a bunch of references to 'DID Doc'). The same would likely need to be done for the DID-Exchange RFC, as this is mainly a maintenance PR until that becomes the standard.